doc/news/DM-43824.bugfix.rst

```diff
@@ -0,0 +1 @@
+Add descriptions for the two OCPS events.
```

Comment:
In a way this indeed is a bugfix but the news item should be in `doc/news/interface_changes/DM-43824.ocps.rst` since it is related to a CSC, as explained in the README.rst file. The `bugfix` and other categories refer to ts_xml bugfixes and other categories only.